partner_shipper/i18n/es.po

```diff
@@ -0,0 +1,122 @@
+# Translation of Odoo Server.
```

Comment:
Please delete i18n folder.
You can recreate it using Export Translation for your module and v12.